### PR TITLE
Accept user name and affiliation while sign up. Fixes #200.

### LINKED ISSUE #200: when users sign up, ask for full information (name, affiliation)

### PR DESCRIPTION
**Pull request by karandesai-96**
This PR modifies the sign up form to ask for First Name, Last Name and Affiliation. First and last name fields are mandatory, while Affiliation is optional.
Fixes #200

### CODE REVIEW

**Comment by karandesai-96** [comment]
@percyliang Please check this if you missed it out yesterday :smile:

File: codalab/apps/web/templates/web/account/signup.html [hunk 1]
```diff
@@ -25,6 +25,18 @@
         <input id="id_login" class="form-control" name="username" placeholder="Username" type="text" value="{{ request.GET.username}}" required>
     </div>
     <div class="form-group">
+        <label for="id_login">First Name:</label>
+        <input id="id_firstname" class="form-control" name="first_name" placeholder="First Name" type="text" value="{{ request.GET.first_name}}" required>
```
Comment (kashizui) [inline codalab/apps/web/templates/web/account/signup.html]: Nit: space before `}}`

File: codalab/apps/web/templates/web/account/signup.html [hunk 2]
```diff
@@ -25,6 +25,18 @@
...
+    </div>
+    <div class="form-group">
+        <label for="id_login">Last Name:</label>
+        <input id="id_lastname" class="form-control" name="last_name" placeholder="Last Name" type="text" value="{{ request.GET.last_name}}" required>
```
Comment (kashizui) [inline codalab/apps/web/templates/web/account/signup.html]: same as above

File: codalab/apps/web/templates/web/account/signup.html [hunk 3]
```diff
@@ -25,6 +25,18 @@
...
+    </div>
+    <div class="form-group">
+        <label for="id_login">Affiliation:</label>
+        <input id="id_affiliation" class="form-control" name="affiliation" placeholder="Affiliation" type="text" value="{{ request.GET.affiliation}}">
```
Comment (kashizui) [inline codalab/apps/web/templates/web/account/signup.html]: same as above

**Comment by kashizui** [comment]
What happens exactly if the user doesn't fill out Affiliation? Does empty string or None get stored in the database row?

File: codalab/apps/web/templates/web/account/signup.html (hunk 1 above)
Comment (karandesai-96) [inline codalab/apps/web/templates/web/account/signup.html]: I'll clean up all of them :smile:

**Comment by karandesai-96** [comment]
For the sake of continuity of information, I'd refer the answer to above question at this [issue comment](https://github.com/codalab/codalab-cli/pull/541#issuecomment-235874175).

File: codalab/apps/web/templates/web/account/signup.html (hunk 1 above)
Comment (karandesai-96) [inline codalab/apps/web/templates/web/account/signup.html]: :white_check_mark: _FIXED_

**Comment by kashizui** [comment]
LGTM
Thanks for your contribution, I will test and merge this later this week.

**Comment by kashizui** [comment]
Functionality looks fine while testing. Please squash your commits into a single commit when you're ready to merge.

**Comment by karandesai-96** [comment]
That's great ! I will do it in the evening, meanwhile if you wish to automatically squash and merge, feel free to do so. ( [Reference](https://github.com/blog/2141-squash-your-commits) )
Thanks.

**Comment by kashizui** [comment]
Oops I didn't know about that feature, thanks for the heads up.

**Pull request merged**